options.go

```diff
@@ -0,0 +1,61 @@
+package boa
+
+import tea "github.com/charmbracelet/bubbletea"
+
+type options struct {
+	// public
+	atlScreen  tea.ProgramOption
```

Comment:
Looks like this should be `altScreen` t and l are swapped.

Reply:
fixed